gallery/src/components/NxCombobox/NxComboboxPage.tsx

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxH3, NxTile, NxTextLink } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxComboboxExample from './NxComboboxExample';
+import NxComboboxPredeterminedListExample from './NxComboboxPredeterminedListExample';
+import NxComboboxDisabledExample from './NxComboboxDisabledExample';
+import NxComboboxValidationExample from './NxComboboxValidationExample';
+import NxComboboxErrorExample from './NxComboboxErrorExample';
+
+const NxComboboxExampleCode = require('./NxComboboxExample?raw');
+const NxComboboxPredeterminedListExampleCode = require('./NxComboboxPredeterminedListExample?raw');
+const NxComboboxDisabledExampleCode = require('./NxComboboxDisabledExample?raw');
+const NxComboboxValidationExampleCode = require('./NxComboboxValidationExample?raw');
+const NxComboboxErrorExampleCode = require('./NxComboboxErrorExample?raw');
+
+const NxComboboxPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A search text input that shows matching results in a dropdown for the user to select.
```

Comment:
This description seems like it was probably copied from `NxSearchDropdown` and doesn't really fit

Reply:
Updated here d0e442dc197dbeaced5824330a1d5864955e5657.